src/Bridges/ApplicationDI/LatteExtension.php

```diff
@@ -19,6 +19,7 @@ class LatteExtension extends Nette\DI\CompilerExtension
 	public $defaults = [
 		'xhtml' => FALSE,
 		'macros' => [],
+		'templateClass' => \Nette\Bridges\ApplicationLatte\Template::class
```

Comment:
* should be simplified to `Nette\Bridges\ApplicationLatte\Template::class`
* the usecase se imho too narrow to add option to the extension so I would remove this entirely
* if added, the default value should be `NULL` and the setter call should only be emitted if the value is not NULL